src/core/src/core_logic/PatchInstaller.py

```diff
@@ -400,36 +410,23 @@ def install_patches_in_batches(self, all_packages, packages, package_versions, m
                 maintenance_window_batch_cutoff_reached = True
                 break
 
```

Comment:
Add comment here to say initally it will be only parent packages. later it will be filled with dependencies too.

Reply:
Done